src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  sourceString.split(';').forEach((line) => {
+    const trimmedLine = line.trim();
+
+    const [property, value] = trimmedLine.split(':');
+
+    if (property && value) {
+      styleObject[property.trim()] = value.trim();
+    }
+  });
+
+  return styleObject;
```

Comment:
U can use reduce method for it
```suggestion
  return sourceString.split(';').reduce((acc, style) => {
    const [property, value] = style.trim().split(':');

    if (property && value) {
      acc[property.trim()] = value.trim();
    }
    
    return acc;
  }, {});
```